ChatSecure/Classes/OTRAppDelegate.m

```diff
@@ -569,7 +569,9 @@ - (void) userNotificationCenter:(UNUserNotificationCenter *)center willPresentNo
 
 - (void) userNotificationCenter:(UNUserNotificationCenter *)center didReceiveNotificationResponse:(UNNotificationResponse *)response withCompletionHandler:(void (^)())completionHandler {
     NSDictionary *userInfo = response.notification.request.content.userInfo;
-    if (userInfo[kOTRNotificationThreadKey] == nil) {
+    if ([kOTRNotificationTypeNone isEqualToString:userInfo[kOTRNotificationType]]) {
+        // Nothing
+    } else if ([kOTRNotificationTypeSubscriptionRequest isEqualToString:userInfo[kOTRNotificationType]]) {
```

Comment:
No, I believe this will work as intended. If userInfo[kOTRNotificationType] is nil the isEqualToString will return NO and we'll fall through to the final else-clause.